_plugins/docs_generator.rb

```diff
@@ -0,0 +1,109 @@
+require 'fileutils'
+require 'nokogiri'
+
+class DocPage < Jekyll::Page
```

Comment:
@apojomovsky consider moving this class to the `_ruby_libs/pages.rb` source file.

Reply:
done, thanks!